cartpole_v0: #namespace

    # Cartpole3D environment variables
    n_actions: 2 # Number of actions used by algorithm and task

    min_pole_angle: -0.7 #-23°
    max_pole_angle: 0.7 #23°
    max_base_velocity: 50
    min_base_pose_x: -1.0
    max_base_pose_x: 1.0
    pos_step: 1.0     # increment in position/velocity/effort, depends on the control for each command
    running_step: 0.04 # amount of time the control will be executed
    init_pos: 0.0 # Position in which the base will start
    wait_time: 0.1 # Time to wait in the reset phases
    control_type: "velocity"



